core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -1215,10 +1214,9 @@ object ReassignPartitionsCommand extends Logging {
   def cancelPartitionReassignments(adminClient: Admin,
                                    reassignments: Set[TopicPartition])
   : Map[TopicPartition, Throwable] = {
-    val results: Map[TopicPartition, KafkaFuture[Void]] =
-      adminClient.alterPartitionReassignments(reassignments.map {
-          (_, (None: Option[NewPartitionReassignment]).asJava)
-        }.toMap.asJava).values().asScala
+    val results = adminClient.alterPartitionReassignments(reassignments.map {
+      (_, (None: Option[NewPartitionReassignment]).asJava)
```

Comment:
While we are here, could we replace `(None: Option[NewPartitionReassignment]).asJava` by `Optional.empty[NewPartitionReassignment]()` or `Option.empty[NewPartitionReassignment].asJava`? Personally, I prefer using `Optional` directly here as we use `Optional.of` above.

Reply:
Good idea.I updated the code, please review it again, thank you.